src/aces/configs/aces.yaml

```diff
@@ -12,6 +12,8 @@ config_path: ${cohort_dir}/${cohort_name}.yaml
 # Path to store the output file. The `${data._prefix}` addition allows us to add shard specific prefixes in a
 # sharded data mode.
 output_filepath: ${cohort_dir}/${cohort_name}${data._prefix}.parquet
+# Optional path to store the output file with the raw window data.
+window_stats_filepath: null
```

Comment:
so, actually, we'll need to make this a touch more complicated -- you need to use `${data._prefix}` to account for sharding, so you get a file per shard instead of one file that overwrites itself as each shard finishes.

I'd suggest just making a `window_stats_dir` variable that defaults to null then have a `window_stats_filename` arg that defaults to `window_stats/${cohort_name}${data._prefix}.parquet` string that can be overwritten, then just put the two together in `__main__` or in the config if that doesn't cause errors when `window_stats_dir` is `null` so the user just needs to overwrite the dir when they want to use it but still get the sharding benefit.